OpenSim/Simulation/SimbodyEngine/Coordinate.h

```diff
@@ -281,15 +281,15 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Coordinate, ModelComponent);
     //       during a simulation.
     //       The last constraint to be set takes precedence.
     /** Indices for the constraint in Simbody. */
-    SimTK::ConstraintIndex _prescribedConstraintIndex;
-    SimTK::ConstraintIndex _lockedConstraintIndex;
-    SimTK::ConstraintIndex _clampedConstraintIndex;
+    SimTK::ResetOnCopy<SimTK::ConstraintIndex> _prescribedConstraintIndex;
+    SimTK::ResetOnCopy<SimTK::ConstraintIndex> _lockedConstraintIndex;
+    SimTK::ResetOnCopy<SimTK::ConstraintIndex> _clampedConstraintIndex;
```

Comment:
I don't see corresponding changes to constructors in Coordinate.cpp. Is that because these were never being manually reset on copy properly before?